services/crypto.proto

```diff
@@ -0,0 +1,6 @@
+message Roster {
```

Comment:
- Added `ServerIdentity`
- changed name to `network.proto`
- as long as it's only used for services, I think the `cothority/services` is a good place for the `network.proto`-file